docs/contributing.md

```diff
@@ -148,120 +173,117 @@ To use a preview build for a package within a project, you need to override the
 
 4. Run `yarn install`.
 
-## Adding new packages
+## Releasing changes
```

Comment:
This section got moved up, and expanded to give people more guidance than before.